lr-cli/cortx_setup/validate.py

```diff
@@ -39,3 +41,26 @@ def path(path):
             raise argparse.ArgumentTypeError(
                 f"cannot access {path}: No such file ")
     return path
+
+
+def interfaces(interface):
+    for iface in interface:
+        try:
+            cmd_run(f"ip a | grep {iface}")
+        except Exception as exc:
+            raise CortxSetupError(f"Invalid interface {iface}\n {exc}")
+
+
+def disk_devices(device_type, devices):
+    local_devices = None
+    if device_type == HW_TYPE:
+        local_devices = cmd_run("multipath -ll|grep mpath|sort -k2|cut -d' ' -f1|sed 's|mpath|/dev/disk/by-id/dm-name-mpath|g'|paste -s -d, -")  # noqa: E501
```

Comment:
will this fail if there are no multipath devices?

Reply:
this command will return empty string, I have handle that case in code.

Reply:
ok..